scripts/process-konflux-prs/check-merge-safety.sh

```
@@ -0,0 +1,66 @@
+            ;;
+    esac
+done
+
+if [[ ${#COMPONENTS[@]} -eq 0 ]]; then
+    echo "Usage: $0 --components 'comp1 comp2 ...'" >&2
+    exit 1
+fi
+
+blocking='[]'
+for component in "${COMPONENTS[@]}"; do
+    result=$("$SCRIPT_DIR/check-pipeline-status.sh" --component "$component" $NAMESPACE_ARG)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)